conversion.go

```diff
@@ -33,3 +39,48 @@ func base64ToBytes(base64Str string) ([]byte, error) {
 func urlSafeBase64ToBytes(urlSafeBase64 string) ([]byte, error) {
 	return base64.URLEncoding.DecodeString(urlSafeBase64)
 }
+
+func bytesToBool(bytes []byte) (result bool, err error) {
+	result, err = strconv.ParseBool(string(bytes))
+	if err != nil {
+		log.Printf(
+			"Unable to parse bytes to bool. Error: %s", err)
+		return false, err
+	}
+
+	return result, nil
+}
+
+func convertAttribute(protoAttribute *yotiprotoattr_v3.Attribute) *attribute.Attribute {
+	processedAnchors := anchor.ParseAnchors(protoAttribute.Anchors)
+
+	var attrType attribute.AttrType
+
+	switch protoAttribute.ContentType {
+	case yotiprotoattr_v3.ContentType_STRING:
+		attrType = attribute.AttrTypeString
+
+	case yotiprotoattr_v3.ContentType_DATE:
+		attrType = attribute.AttrTypeTime
+
+	case yotiprotoattr_v3.ContentType_JPEG:
+		attrType = attribute.AttrTypeJPEG
+
+	case yotiprotoattr_v3.ContentType_PNG:
+		attrType = attribute.AttrTypePNG
+
+	case yotiprotoattr_v3.ContentType_JSON:
+		attrType = attribute.AttrTypeJSON
+
+	case yotiprotoattr_v3.ContentType_UNDEFINED:
+	default:
+		attrType = attribute.AttrTypeInterface
+	}
+
+	return &attribute.Attribute{
+		Name:    protoAttribute.Name,
+		Value:   protoAttribute.Value,
+		Type:    attrType,
+		Anchors: processedAnchors,
+	}
+}
```

Comment:
What is the value of this re-typing?

Reply:
Were you thinking that we should just keep the protobuf types throughout, and not convert them? I guess it provides us with a cleaner attribute object, which doesn't have the extra fields which the protobuf attributes have.

Reply:
Yes, I think that using the proto throughout will mean less to maintain and no data/fidelity loss in translation. I initially suggested embedding it, but I think just have use an internal field.

Reply:
I've created [SDK-624](https://lampkicking.atlassian.net/browse/SDK-624) to look at this, but am going to keep it like this for now.

Reply:
I've addressed this in a commit to this branch now - https://github.com/getyoti/yoti-go-sdk/pull/22/commits/f412515397e65b30a07873d872dc0c09415cb4f0